extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -108,15 +106,29 @@ function PanelStudyBrowserTracking({
     };
   }, [measurementService, activeViewportIndex, sendTrackedMeasurementsEvent]);
 
-  const { trackedStudy, trackedSeries } = trackedMeasurements.context;
+  const { trackedSeries } = trackedMeasurements.context;
 
   // ~~ studyDisplayList
   useEffect(() => {
     // Fetch all studies for the patient in each primary study
     async function fetchStudiesForPatient(StudyInstanceUID) {
-      const qidoStudiesForPatient =
-        (await getStudiesForPatientByStudyInstanceUID(StudyInstanceUID)) || [];
-      // TODO: This should be "naturalized DICOM JSON" studies
+      // current study qido
```

Comment:
Is there any way we can create a method somewhere so that we can avoid duplicating this code?

Reply:
as discussed internally, we will do this later